client/importServiceWorker.js

```diff
@@ -59,21 +75,15 @@ Meteor.startup(() => {
 							if (sub === null) {
 								console.log('Not subscribed to push service!');
 								if (settingsReady) {
-									modal.open({
```

Comment:
@bhardwajaditya @ear-dev If we need this modal anyhow, then can we do something like this top banner?
Because showing modal blocking whole screen does look good in UX perspective.
let me know your thoughts

![localhost_3000_home(Pixel 2)](https://user-images.githubusercontent.com/18264684/93117527-a03fdf80-f6dc-11ea-826b-38f951cf10a4.png)

Reply:
What we planned initially was push notifications won't be optional for users it's required for PWA. Being a chat application notifications plays an important part.  Even on android apps we get so many modals at startup asking for basic permissions, that UX was following the similar pattern. If we do this top banner only half the PWA users will opt for notifications and that will degrade the overall chat experience

Reply:
I have re-added the modal 👍